### ISSUE TITLE
JS error on start: ambiguous indirect export: ClassicEditor

### ISSUE THREAD

**Issue by a-schild** [wontlost-ltd/vaadin-litelement-ckeditor#39]
I did just upgrade from a 23.1.x to 23.3.0 vaadin flow project.
And of course also did upgrade ckeditor to 3.1.5

Vite compilation seems to work, but when the app starts in dev mode, I see this in the browser console:
ambiguous indirect export: ClassicEditor

And then the loading stops.

I see two files with `ClassicEditor`in them,
`generated/ckeditor.js`
and
`generated-vaadin-ckeditor.js`

Any ideas?

**Comment by a-schild** [comment]
Full stack trace when trying to build a production release:

```
[ERROR] [36mvite v3.2.4 building for production...
[ERROR] transforming...
[ERROR] Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
[ERROR] Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
[ERROR] â?? 949 modules transformed.
[ERROR] 'ClassicEditor' is not exported by frontend/generated/jar-resources/ckeditor.js, imported by frontend/generated/jar-resources/vaadin-ckeditor.js
[ERROR] file: c:/Develop/Boost.Git/bas-frontend/frontend/generated/jar-resources/vaadin-ckeditor.js:3:23
[ERROR] 1: import {html, LitElement} from 'lit';
[ERROR] 2: import {classMap} from 'lit/directives/class-map.js';
[ERROR] 3: import {BalloonEditor, ClassicEditor, DcoupledEditor, InlineEditor} from './ckeditor';
[ERROR]                           ^
[ERROR] 4:
[ERROR] 5: export class VaadinCKEditor extends LitElement {
[ERROR]
[ERROR] error during build:
[ERROR] Error: 'ClassicEditor' is not exported by frontend/generated/jar-resources/ckeditor.js, imported by frontend/generated/jar-resources/vaadin-ckeditor.js
[ERROR]     at error (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:1858:30)
[ERROR]     at Module.error (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:12429:16)
[ERROR]     at Module.traceVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:12788:29)
[ERROR]     at ModuleScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:11440:39)
[ERROR]     at ChildScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
[ERROR]     at ClassBodyScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
[ERROR]     at ChildScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
[ERROR]     at FunctionScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
[ERROR]     at ChildScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)
[ERROR]     at TrackingScope.findVariable (file:///c:/Develop/Boost.Git/bas-frontend/node_modules/rollup/dist/es/shared/rollup.js:6372:38)

```

**Comment by eroself** [comment]
Thanks for telling me this issue. Will look into this soon. Cheers.

**Comment by a-schild** [comment]
Would be great.

When I remove `ClassicEditor`from the import, it struggles on the next one: `DcoupledEditor`

**Comment by eroself** [comment]
Should have been fixed in version v3.2.0

**Comment by a-schild** [comment]
Works fine.
Thanks for the timely fix.
